features/check_adds_functions.feature

```diff
@@ -0,0 +1,32 @@
+Feature: These are functions that add two numbers
+  
+  Scenario: We should have the correct resources for adding two numbers
+    Given there is 4 function that can add two numbers
+    
+  Scenario: A person wants to add two numbers together using Kendell's Function
+    Given we have a function that can add two numbers called kendellsAddNumberFunction
+      And we want to use 134872478 as the first number
+      And we want to use 5230 as the second number
+    When we add our numbers using the kendellsAddNumberFunction function
+    Then the result should equal 134877708
+    
+  Scenario: A person wants to add two numbers together using Zach's Function
+    Given we have a function that can add two numbers called zachmultiplyTwoNumbersandsubtractnumberTwo
+      And we want to use 3479225 as the first number
+      And we want to use 45432345 as the second number
+    When we add our numbers using the zachmultiplyTwoNumbersandsubtractnumberTwo function
+    Then the result should equal 48911570
+    
+  Scenario: A person wants to add two numbers together using Matt's Function
+    Given we have a function that can add two numbers called mattsDestroyNumbersFunction
+      And we want to use 1 as the first number
+      And we want to use 2 as the second number
+    When we add our numbers using the mattsDestroyNumbersFunction function
+    Then the result should equal 2
+    
+  Scenario: A person wants to add two numbers together using Frank's Function
```

Comment:
Since the function in question is not written to the original parameter, this single test scenario is the only one under which the function can pass the test. Function needs rewriting in order to pass test in all scenarios.